src/compiler/crystal/codegen/debug.cr

```diff
@@ -335,8 +335,11 @@ module Crystal
       if basic_block
         block = basic_block
       else
-        basic_blocks = context.fun.basic_blocks.to_a
-        block = basic_blocks.size == 0 ? new_block("alloca") : basic_blocks[-1]
+        block = nil
+        context.fun.basic_blocks.each do |current_block|
+          block = current_block
+        end
```

Comment:
```suggestion
        block = context.fun.basic_blocks.last?
```

Reply:
Error: undefined method 'last?' for LLVM::BasicBlockCollection

Reply:
Yeah, just noticed it's defined on `Indexable` instead of `Enumerable` as I've initially thought... that's weird though.

Reply:
Well, this can re-evaluated once/if #8959 gets merged.

Reply:
I just moved that code into BasicBlocksCollection and named it last?

So now my code is:
```
        block = context.fun.basic_blocks.last? || new_block("alloca")
```